nextflow.config

```diff
@@ -235,6 +233,108 @@ manifest {
     doi             = ''
 }
 
+validation {
+    monochromeLogs = false
+    // Use --help, --help <parameter> or --helpFull to show a help message
+    help {
+        enabled = true
+        beforeText = nfCoreLogo()
+        command = "nextflow run ${manifest.name} -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
+        afterText = workflowCitations()
+    }
+}
+
+def workflowCitations() {
```

Comment:
Will we be able to define these function in a NF script instead of the config? Since code is not advised in config files and it might be disabled in the future.